src/topology/algebra/order/basic.lean

```diff
@@ -980,8 +980,27 @@ lemma tendsto_nhds_bot_mono' [topological_space β] [partial_order β] [order_bo
 tendsto_nhds_bot_mono hf (eventually_of_forall hg)
 
 section linear_order
+variables [topological_space α] [linear_order α]
 
-variables [topological_space α] [linear_order α] [order_topology α]
+section order_closed_topology
+variables [order_closed_topology α] {a b : α}
+
+lemma eventually_le_nhds (hab : a < b) : ∀ᶠ (x : α) in (nhds a), x ≤ b :=
```

Comment:
```suggestion
lemma eventually_le_nhds (hab : a < b) : ∀ᶠ x in 𝓝 a, x ≤ b :=
```
Purely cosmetic of course!

Reply:
True! In another file I was working in I hadn't opened the right locale for the fancy N. Will change here for consistency with this file